core/stores/sqlx/sqlconn.go

```diff
@@ -303,9 +316,11 @@ func (s statement) Exec(args ...interface{}) (sql.Result, error) {
 	return s.ExecCtx(context.Background(), args...)
 }
 
-func (s statement) ExecCtx(ctx context.Context, args ...interface{}) (sql.Result, error) {
-	ctx, span := startSpan(ctx)
-	defer span.End()
+func (s statement) ExecCtx(ctx context.Context, args ...interface{}) (result sql.Result, err error) {
```

Comment:
it be better to return anonymous values？result and err are not used

Reply:
he defer call depends on the err value. the err might be changed later.